public/js/components/RightSidebar.css

```diff
@@ -13,8 +13,8 @@
 .command-bar > span {
   cursor: pointer;
   margin-right: 0.7em;
-  width: 1em;
-  height: 1.1em;
+  width: 16px;
```

Comment:
I don't think so; they are icons and should not be resized with the font. The command bar has a fixed height of 30px so the icons should fit inside of it exactly. I don't see the benefits of ems here.